src/main/resources/application-local.yaml

```diff
@@ -6,27 +6,21 @@ spring:
   boot:
     admin:
       ui:
-        hide-instance-url: true
+        hide-instance-url: false  # Show URLs in local development
       hazelcast:
         start-embedded: false
 
 eureka:
   client:
     enabled: true
     serviceUrl:
-      defaultZone: http://localhost:8761/eureka/
```

Comment:
the purpose of this `local` spring profile is to run this component using `mvn spring-boot:run -Dspring-boot.run.profiles=local` and not trough a docker image, the default profile of this application is optimal for docker already.

Reply:
local means = running in the host jvm
docker is not considered local from this perspective